go.mod

```diff
@@ -35,6 +35,7 @@ require (
 	cloud.google.com/go/compute/metadata v0.2.3 // indirect
 	cloud.google.com/go/iam v1.1.1 // indirect
 	cloud.google.com/go/kms v1.12.1 // indirect
+	github.com/DataDog/zstd v1.4.5 // indirect
```

Comment:
Why is datadog added here? Is this adding some sort of reporting?

Reply:
It is an indirect dependency from some direct dependency:
```
github.com/DataDog/zstd
This module is necessary because [github.com/DataDog/zstd](vscode-file://vscode-app/snap/code/143/usr/share/code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) is imported in:

- github.com/onflow/flow-cli/internal/accounts
-- github.com/onflow/flow-cli/flowkit/gateway
--- github.com/onflow/flow-go/model/flow
---- github.com/onflow/flow-go/model/flow.test
----- github.com/onflow/flow-go/utils/unittest
------ github.com/cockroachdb/pebble
------- github.com/cockroachdb/pebble/sstable
```